scripts/delete.js

```diff
@@ -0,0 +1,30 @@
+import { getTasksFromStorage, saveTaskToStorage } from "./add.js";
+
+document.getElementsByClassName("deleteEditButton")[0].addEventListener("click", () => {deleteTaskById(document.getElementsByClassName("deleteEditButton")[0].id)});
+
+/**
+ * Reads 'to-do-tasks' from localStorage, removes the task with the given ID
+ * And then rewrites the updated list of Tasks back to local storage.
+ * Then, it reloads the page to refresh the task list
+ * @param {number} id ID of the task to be removed from local storage
+ */
+export function deleteTaskById(id){
+
+    const taskList = getTasksFromStorage();
+    
+    //const del = document.getElementById('id');
+
+    for(var i = 0; i < taskList.length; i++) {
+        /**if(taskList[i] == del) {
```

Comment:
Please delete the commented out code, as mentioned by Prof it is a good habit to remove commented out code

Reply:
I agree, unless we might use it in future (same for line 15)